amd/src/mutations.js

```diff
@@ -190,11 +196,20 @@ export default class {
                 boardid: state.board.id,
                 data: data
             },
+            fail: this.processFail,
```

Comment:
could also be done by using try { } catch () ..., up to you :)

Reply:
This one is shorter ;)